modules/util/util.js

```diff
@@ -542,3 +542,65 @@ export function utilUnicodeCharsCount(str) {
 export function utilUnicodeCharsTruncated(str, limit) {
     return Array.from(str).slice(0, limit).join('');
 }
+
+// Simple polyfill for String.prototype.padStart().
+function padStart(str, width, pad) {
+    var padded = Array(width);
+    padded.fill(pad);
+    padded = padded.join('');
+    return padded.substr(0, width - str.length) + str;
+}
```

Comment:
PhantomJS throws an error when using `String.prototype.padStart()`, probably because this fork of iD is based on an older mainline iD from before it dropped support for IE for openstreetmap#8811.

Reply:
After rebasing onto the latest staging branch, tests pass even when using the built-in `String.prototype.padStart()`.